src/framework/draw/types/font.h

```diff
@@ -33,6 +33,30 @@ namespace muse::draw {
 class Font
 {
 public:
+    struct FontFamily {
+        FontFamily() = default;
+        FontFamily(const char* id)
+            : m_id(String::fromUtf8(id)) {}
```

Comment:
Maybe make this `const char16_t*` instead, to avoid the String::fromUtf8 conversion